### DISCUSSION TITLE
Velero - Unable to restore PersistentVolumes due to x509 Errors

### DISCUSSION THREAD

**Discussion by boluisa** [vmware-tanzu/velero#3677] [Community support Q&A] [answered]
Hi all!

We are facing this issue with velero restore when passing in ca-cert for communicating with cloud provider.  Backups are completing successfully but restores are not.

Any help on this would be highly appreciated.

Environment

Velero 1.5.4
Kubernetes 1.18

What did I do?

**Issue**
Restores fail with the following approximate error:
```
error executing PVAction for persistent volumes
Caused by POST https://ec2.region.gov    x509 certificate signed by unknown authority
```

We're using the following install command, using the velero CLI, to create a YAML manifest which we edit slightly to add in additional proxy environment variables to velero.

```
velero install --dry-run \
               --output yaml \
               --provider aws \
               --bucket velero-backups \
               --backup-location-config region=us-gov-west-1 \
               --snapshot-location-config region=us-gov-west-1 \
               --image <velero v.1.5.3> \
               --plugins  <velero-plugin-for-aws:v1.2.0> \
               --no-secret \
               --prefix blue-k8s-cluster \
               --cacert /path/to/private/bundle.crt > velero.yaml
```

**Comment by carlisia** [comment, marked as answer]
I'm going to mark this as "answered" here because we will be tracking this as an issue: https://github.com/vmware-tanzu/velero/issues/3678
